QuoteOfTheDayQuickStart/README.md

```diff
@@ -0,0 +1,374 @@
+# QuoteOfTheDayQuickStart
```

Comment:
I hope we can a more user-friendly name for the article, for example, Build the "Quote of the Day" application